Traceback (most recent call last):
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
print('C:\Albert Gator\URP6271\class_data.gdb\major_roads')
------------------

[1;36m  File [1;32m"<ipython-input-5-7d4c6a137953>"[1;36m, line [1;32m1[0m
[1;33m    print('C:\Albert Gator\URP6271\class_data.gdb\major_roads')[0m
[1;37m                                                              ^[0m
[1;31mSyntaxError[0m[1;31m:[0m (unicode error) 'unicodeescape' codec can't decode bytes in position 15-16: truncated \UXXXXXXXX escape

SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 15-16: truncated \UXXXXXXXX escape (<ipython-input-5-7d4c6a137953>, line 1)

